common/lib/util/forInOwnNonNullProps.ts

```diff
@@ -0,0 +1,7 @@
+export default function(ob: Record<string, unknown>, fn: Function) {
```

Comment:
For an implementation that literally uses a fully qualified / expanded name like [hasOwnProperty](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwnProperty), I'm confused why we're abbreviating the word 'properties'.

While I recognise that there are frameworks like React which have popularised `Props` as a shortening, if there's no strong reason why we need it shorter then I think it would be clearer for those reading the code if this file were called `forInOwnNullProperties` and it were imported as such where needed.

Reply:
Agreed, 33a6451.